xchange-bittrex/src/main/java/org/knowm/xchange/bittrex/BittrexExchange.java

```diff
@@ -70,7 +76,16 @@ public void remoteInit() throws IOException {
       synchronized (INIT_LOCK) {
         if (bittrexSymbols.isEmpty()) {
           bittrexSymbols = ((BittrexMarketDataServiceRaw) marketDataService).getBittrexSymbols();
-          BittrexAdapters.adaptMetaData(bittrexSymbols, exchangeMetaData);
+          List<BittrexCurrency> bittrexCurrencies =
+                  ((BittrexMarketDataServiceRaw) marketDataService).getBittrexCurrencies();
+          Map<CurrencyPair, Fee> dynamicTradingFees = null;
+          try {
+            dynamicTradingFees = accountService.getDynamicTradingFees();
+          } catch (BittrexException | IOException e) {
+            Logger.getLogger(BittrexExchange.class.getName()).log(Level.WARNING,
```

Comment:
please create a static object like is done in the rest of the codebase

Reply:
I don't understand the comment here. Can you give a little bit more detail on the comment ? Which one should be static ? Logger.getLogger result ? or something else ?